bangazonapi/views/profile.py

```diff
@@ -156,7 +156,7 @@ def cart(self, request):
 
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False, context={'request': request}).data
-                cart["order"]["line_items"] = line_items.data
+                # cart["order"]["line_items"] = line_items.data
```

Comment:
did you leave this commented code in here on purpose?

Reply:
Removing that line is the solution. I wasn't sure if permanently removing it was the proper way to handle it, so I just commented it out. I can remove it though.